src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { EnrichPerson, Person } from '../types';
+import { Loader } from './Loader';
+import { PeopleTable } from './PeopleTable';
+
+const enrichPeople = (people: Person[]): EnrichPerson[] => {
+  return people.map(person => {
+    const mother = people.find(pers => pers.name === person.motherName) || null;
+    const father = people.find(pers => pers.name === person.fatherName) || null;
+
+    return { ...person, mother, father };
+  });
+};
+
+export const PeoplePage = () => {
+  const [people, setpeople] = useState<Person[]>([]);
```

Comment:
Consider renaming `setpeople` to `setPeople` to follow the camelCase convention typically used in JavaScript and TypeScript for function names.